Core/scripts/dirac-create-distribution-tarball.py

```diff
@@ -463,10 +458,9 @@ def __compileReleaseNotes( self, rstFile ):
     for baseFileName in baseList:
       htmlFileName = baseFileName + ".html"
       try:
-        fd = open( htmlFileName, "w" )
-        fd.write( parts[ 'whole' ] )
-        fd.close()
-      except Exception, excp:
+        with open( htmlFileName, "w" ) as fd:
+          fd.write( parts[ 'whole' ] )
+      except Exception as excp:
         return S_ERROR( "Could not write %s: %s" % ( htmlFileName, excp ) )
```

Comment:
(Guessing this is a response to my comment)
Using the repr() function instead of str() will print more information, which might be enough?
```
In [2]: x = KeyError(3)
In [3]: repr(x)
Out[3]: 'KeyError(3,)'
```
(also via using %r instead of %s )